### PR TITLE
Updates to OPTIONS and TRACE findings

### PR DESCRIPTION
**Pull request by RRudder**
The following updates have been made to the OPTIONS and TRACE method findings:
Standardised grammar and flow of findings
Removed JSON guidance from within the templates.md file and created a seperate file
Added a recommendations.md file for future use and drafted some content.

### CODE REVIEW

**Comment by RRudder** [comment]
The recommendation for these findings is a little vague. I was unsure if there needed to be more detail about how to disable the use of these HTTP methods.

File: submissions/description/server_security_misconfiguration/potentially_unsafe_http_method_enabled/options/guidance.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Guidance
+
+<!--
+Provide a 1-2 sentence description - see http://cveproject.github.io/docs/content/key-details-phrasing.pdf for tips.
+Also provide a step-by-step walkthrough with screenshot on how you exploited the vulnerability. This will speed triage time and result in faster rewards!
+
+Your submission MUST include evidence of the vulnerability and not be theoretical in nature.
+Attempt to escalate the vulnerability to perform additional actions. If this is possible, provide a full proof-of-concept.
+-->
```
Comment (codingo) [inline submissions/description/server_security_misconfiguration/potentially_unsafe_http_method_enabled/options/guidance.md:9]: HTML comment to be removed

Comment (RRudder) [reply submissions/description/server_security_misconfiguration/potentially_unsafe_http_method_enabled/options/guidance.md:9]: Thank you, I missed these tags. Will remove now

**Comment by RRudder** [comment]
Closing as more recent PR update includes up to date changes and no conflicts

**Pull request closed without merging**